app/src/main/java/chat/rocket/android/api/MethodCallHelper.java

```diff
@@ -462,15 +466,43 @@ public MethodCallHelper(RealmHelper realmHelper, DDPClientRef ddpClientRef) {
   }
 
   public Task<Void> searchSpotlightUsers(String term) {
-    return searchSpotlight(
-        RealmSpotlightUser.class, "users", term
-    );
+    return searchSpotlight(RealmSpotlightUser.class, "users", term);
   }
 
   public Task<Void> searchSpotlightRooms(String term) {
-    return searchSpotlight(
-        RealmSpotlightRoom.class, "rooms", term
-    );
+    return searchSpotlight(RealmSpotlightRoom.class, "rooms", term);
+  }
+
+  public Task<Void> searchSpotlight(String term) {
```

Comment:
I'd advocate to try to not spread more `Task`'s on the code base... Since this is a `Void` task, it could return a `Completable` and then it would be easy to compose a stream on the outer layers (Presenters, UI, etc).

On the future, I'd like to get rid of bolts tasks in favor of RxJava, so if we don't expose then, it will be easier to accomplish this.